pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/route/AdminLookupInvitationRoute.java

```diff
@@ -63,12 +63,16 @@ public Object handle(Request request, Response response, LookupInvitationPayload
                 }
 
                 String loginEmail = null;
-                if (user.getAuth0UserId() != null) {
+                
+                if (user.getAuth0UserId().isPresent()) {
```

Comment:
Regarding returning optionals.
Not sure, but if we'll define the method `hasAuth0UserId` the code will look like
```
if (user.hasAuth0UserId()) {
  ... = user.getAuth0UserId(); //without additional .get statement.
}
```

Reply:
Didn't notice that method! Did a bit of refactoring.